packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -184,6 +198,7 @@ export class IdentityHeader extends React.PureComponent<Props, State> {
      to have it all in the same place. Also, it is down here as openBackupModal and openForgetModal need
      to be initialized first else tsc will complain.
    */
+  renameTrigger = <Dropdown.Item closeOnFocus icon='edit' onClick={this.openRenameModal} text='Rename Account' />;
```

Comment:
Not sure how it's related to this PR, but I have a new error `Warning: React does not recognize the `closeOnFocus` prop on a DOM element.`

Reply:
hmm closeOnFocus isn't a Dropdown.Item prop, I'll remove it